File: app/src/main/java/can/siempredelao/f1kotlin/backend/StandingsList.java
package can.siempredelao.f1kotlin.backend;

import com.google.gson.annotations.SerializedName;

import java.util.List;

public class StandingsList {

    public String               season;
    public String               round;
    @SerializedName("DriverStandings")
    public List<DriverStanding> driverStandings;

}

File: app/src/main/kotlin/can/siempredelao/f1kotlin/backend/model/StandingsList.kt
package can.siempredelao.f1kotlin.backend.model

import com.google.gson.annotations.SerializedName

class StandingsList(val season: String,
                    val round: String,
                    @SerializedName("DriverStandings") val driverStandings: List<DriverStanding>)
